library/pem.c

```diff
@@ -478,6 +478,9 @@ int mbedtls_pem_write_buffer( const char *header, const char *footer,
     *p++ = '\0';
     *olen = p - buf;
 
+     /* Clean any remaining data previously written to the buffer */
+    memset( buf + *olen, 0, buf_len - *olen );
```

Comment:
Use the `mbedtls_platform_zeroize` instead of `memset`.

Reply:
This is part of an output buffer, so `memset` is ok. When `zeroize` is not necessary, `memset` is preferred because it's more efficient (the compiler knows how to optimize it for code size or speed).